src/pal/inc/pal.h

```diff
@@ -4679,7 +4679,7 @@ enum {
 //
 typedef struct _RUNTIME_FUNCTION {
     DWORD BeginAddress;
-#ifdef _AMD64_
+#if RUNTIME_FUNCTION_HAS_ENDADDRESS
```

Comment:
Now that we have the EndAddress on amd64 only again, could you please revert the change here and get rid of the RUNTIME_FUNCTION_HAS_ENDADDRESS?